app/views/proposals/_form.html.erb

```diff
@@ -18,7 +18,7 @@
     <div class="small-12 column">
       <%= f.label :summary, t("proposals.form.proposal_summary") %>
       <p class="note"><%= t("proposals.form.proposal_summary_note") %></p>
-      <%= f.text_area :summary, rows: 4, maxlength: 200, label: false,
+      <%= f.text_area :summary, rows: 4, maxlength: 350, label: false,
```

Comment:
Yeah it is. From pg manual a `text` field doesn't have a limit of characters (well, a field can contain at most 1GB)